src/modules/cache-client/cache-client.service.ts

```diff
@@ -82,210 +78,172 @@ export class CacheClient implements ICacheClient {
   }
 
   async getRoleDefinition(namespace: string) {
-    return await this.makeRetryRequest(async () => {
-      const { data } = await this._httpClient.get<IRole>(`/role/${namespace}`);
-      return data?.definition;
-    });
+    const { data } = await this._httpClient.get<IRole>(`/role/${namespace}`);
+    console.dir(data, { depth: 3, colors: true });
```

Comment:
It would be better to use `getLogger` and use the logger here if we would like to log the result.

Reply:
sorry, I missed that after debugging